rpm/polkit.spec

```diff
@@ -6,49 +6,79 @@ License:    LGPLv2+
+Patch0027: patches/debian/0.113/Fix-a-possible-NULL-dereference.patch
+Patch0028: patches/debian/0.113/Remove-a-redundant-assignment.patch
+Patch0029: patches/debian/0.113/Fix-duplicate-GError-use-when-uid-is-missing.patch
+Patch0030: patches/debian/0.113/Fix-a-crash-when-two-authentication-requests-are-in-.patch
+Patch0031: patches/debian/0.113/CVE-2015-4625-Use-unpredictable-cookie-values-keep-t.patch
+Patch0032: patches/debian/0.113/CVE-2015-4625-Bind-use-of-cookies-to-specific-uids.patch
+Patch0033: patches/debian/0.113/docs-Update-for-changes-to-uid-binding-Authenticatio.patch
+Patch0034: patches/debian/0.113/Fix-a-per-authorization-memory-leak.patch
+Patch0035: patches/debian/0.113/Fix-a-memory-leak-when-registering-an-authentication.patch
+Patch0036: patches/debian/0.113/CVE-2015-3255-Fix-GHashTable-usage.patch
+Patch0037: patches/debian/0.113/Fix-use-after-free-in-polkitagentsession.c.patch
+Patch0038: patches/debian/0.113/README-Note-to-send-security-reports-via-DBus-s-mech.patch
+Patch0039: patches/debian/0.114/Fix-multi-line-pam-text-info.patch
+Patch0040: patches/debian/0.114/Refactor-send_to_helper-usage.patch
+Patch0041: patches/debian/0.114/Add-gettext-support-for-.policy-files.patch
+Patch0042: patches/debian/0.114/gettext-switch-to-default-translate-no.patch
+Patch0043: patches/debian/0.114/Support-polkit-session-agent-running-outside-user-session.patch
+Patch0044: patches/debian/0.115/Fix-CVE-2018-1116-Trusting-client-supplied-UID.patch
+Patch0045: patches/debian/0.116/Possible-resource-leak-found-by-static-analyzer.patch
+Patch0046: patches/debian/0.116/Elaborate-message-printed-by-polkit-when-disconnecting-fr.patch
+Patch0047: patches/debian/0.116/Error-message-raised-on-every-systemctl-start-in-emergenc.patch
+Patch0048: patches/debian/0.116/Fix-a-critical-warning-on-calling-polkit_permission_new_s.patch
+Patch0049: patches/debian/0.116/Allow-negative-uids-gids-in-PolkitUnixUser-and-Group-obje.patch
+Patch0050: patches/debian/0.116/tests-add-tests-for-high-uids.patch
+Patch0051: patches/debian/0.116/backend-Compare-PolkitUnixProcess-uids-for-temporary-auth.patch
+Patch0052: patches/debian/0.116/Allow-uid-of-1-for-a-PolkitUnixProcess.patch
+Patch0053: patches/debian/0.116/pkttyagent-PolkitAgentTextListener-leaves-echo-tty-disabl.patch
+Patch0054: patches/debian/01_pam_polkit.patch
+Patch0055: patches/debian/02_gettext.patch
+Patch0056: patches/debian/05_revert-admin-identities-unix-group-wheel.patch
+Patch0057: patches/debian/06_systemd-service.patch
+Patch0058: patches/debian/10_build-against-libsystemd.patch
+Patch0059: patches/debian/Move-D-Bus-policy-file-to-usr-share-dbus-1-system.d.patch
+Patch0060: patches/debian/Statically-link-libpolkit-backend1-into-polkitd.patch
+Patch0061: patches/debian/Remove-example-null-backend.patch
+Patch0062: patches/debian/CVE-2021-3560.patch
+Patch0063: patches/0003-Support-for-annotation-identity-group-check.patch
```

Comment:
Why these three are backwards? 0003 first and then 0002 and 0001.

Reply:
Because of Debian patch ordering, it doesn't apply properly in a different order.

Reply:
Ok, just seemed confusing to me.

Reply:
But these are non-debian patches and used to be in numerical order? If we want the order changed, could then also update the file name numbering?

Reply:
> But these are non-debian patches and used to be in numerical order? If we want the order changed, could then also update the file name numbering?
Fixed that.